### ISSUE TITLE
Possible to retrieve data

### ISSUE THREAD

**Issue by EvertJob** [lostfields/python-panasonic-comfort-cloud#21]
Hi,

In a few weeks my new Panasonic airco will be mounted. Is it possible te retrieve any data from the AC and get it as output or if this plugin just for setting the settings?

Thanks!

**Comment by LudovicRousseau** [comment]
You can get the current data, and also the historical data.
See https://github.com/lostfields/python-panasonic-comfort-cloud/pull/11 for example.

@lostfields maybe you should add some output samples in the README.md or in a wiki page.

In my case I get the current data every 30 minutes and inject them in an [influxdb](https://www.influxdata.com/) database connected to a [grafana](https://grafana.com/) for display.

**Comment by EvertJob** [comment]
That is exactly what I would like (InfluxDB), do you have the code for this public available or any push in the right direction? Why invent the wheel a second time ;)

**Comment by LudovicRousseau** [comment]
You can get my code in my fork in the branch influxdb directory influxdb
https://github.com/LudovicRousseau/python-panasonic-comfort-cloud/tree/influxdb/influxdb

**Comment by EvertJob** [comment]
Great, will check it out when the AC arrived. Looks kinda simple? Just edit the config and run the script I guess?

**Comment by LudovicRousseau** [comment]
Yes.
The code was for me so you may need adapt it to your needs.

**Comment by EvertJob** [comment]
Great. Don’t think i will need a lot, just the energy usage and temperature to begin with :)

**Comment by EvertJob** [comment]
@LudovicRousseau busy with the script and I'm getting some data to Grafana:
`Clim consigne=30.000000,inside=126
`

Only no data like energy usage and kind like that. How can I push that to Grafana?

**Comment by LudovicRousseau** [comment]
Maybe it is because of this line https://github.com/LudovicRousseau/python-panasonic-comfort-cloud/blob/influxdb/influxdb/clim2influxdb.py#L114

I don't know why all your averageInsideTemp are at -255.0.

**Comment by EvertJob** [comment]
> Maybe it is because of this line https://github.com/LudovicRousseau/python-panasonic-comfort-cloud/blob/influxdb/influxdb/clim2influxdb.py#L114
> 
> I don't know why all your averageInsideTemp are at -255.0.

Yes, all ready figured it out. On this moment I use the "averageSettingTemp" to get the error fixed. Why my inside temperatures are empty I don't know. In the app it's the same.

Than the next issue (currently fixed it with a separated SH file) by clim2influxdb_day.sh is not working. I've already changed the rights for the file (the same as clim2influxdb.sh) and I don't get any result.

When I run the file the arguments are just --current instead of the day before.

To solve the error I made a copy of the original clim2influxdb.sh and changes the "ARGS="--current"" to "ARGS="--day $YESTERDAY"" and put this one above: YESTERDAY=`TZ=GMT+24 date +%Y/%m/%d`;
